chart/values.schema.json

```diff
@@ -568,6 +601,23 @@
             "type": "array",
             "x-docsSection": "Airflow",
             "default": [],
+            "items": {
+                "type": "object",
+                "properties": {
+                    "envName": {
+                        "description": "The name of the environment variable under which the secret will be available",
+                        "type": "string"
+                    },
+                    "secretName": {
+                        "description": "The name of the Kubernetes secret that will be read",
+                        "type": "string"
+                    },
+                    "secretKey": {
+                        "description": "The key of the Kubernetes secret",
+                        "type": "string"
+                    }
+                }
```

Comment:
```suggestion
                },
                "additionalProperties": False,
                "required": ["envName", "secretName"]
```

Reply:
Thanks. I also checked and ` "additionalProperties": false,` was missing from the other definitions as well.